client/app/app.js

```diff
@@ -75,7 +74,7 @@ angular
     domainEmailObfuscation,
     domainOptin,
     navbar,
-  ])
+  ].filter(value => value !== null)) // Remove null because __NG_APP_INJECTIONS__ can be null
```

Comment:
```suggestion
  ].filter(Boolean) // Remove null because __NG_APP_INJECTIONS__ can be null
```
will remove all `falsy` values, don't know which one you think is better